rules.d/50-udev-default.rules.in

```diff
@@ -89,4 +89,7 @@ KERNEL=="udmabuf", GROUP="kvm"
 
 SUBSYSTEM=="ptp", ATTR{clock_name}=="KVM virtual PTP", SYMLINK += "ptp_kvm"
 
+SUBSYSTEM=="dmi", ENV{ID_SYSFS_ATTRIBUTE_MODEL}=="", ENV{ID_VENDOR}="$attr{sys_vendor}", ENV{ID_MODEL}="$attr{product_name}"
+SUBSYSTEM=="dmi", ENV{ID_SYSFS_ATTRIBUTE_MODEL}=="?*", ENV{ID_VENDOR}="$attr{sys_vendor}", ENV{ID_MODEL}="$attr{product_version}"
```

Comment:
ideally we'd support `$attr{$env{ID_SYSFS_ATTRIBUTE_MODEL}}`, but given that we don't I think the second line should be:

```
SUBSYSTEM=="dmi", ENV{ID_SYSFS_ATTRIBUTE_MODEL}=="product_version", ENV{ID_VENDOR}="$attr{sys_vendor}", ENV{ID_MODEL}="$attr{product_version}"
```

Given that there aren't a million attributes that are candidates to read here I think we can get away with doing it like this, even if that means we need further rules should some vendor decide to misuse another DMI field for this